ts/src/handlers.ts

```diff
@@ -565,17 +601,27 @@ export class Handlers {
             );
         }
 
+        // Compute approval expiration time and assert transaction expiration
         const approvalExpirationTimeSeconds =
             utils.getCurrentTimestampSeconds() + this._configs.EXPIRATION_DURATION_SECONDS;
+        if (signedTransaction.expirationTimeSeconds.gt(approvalExpirationTimeSeconds)) {
```

Comment:
This should probably be `gte` right?

Reply:
Hmm, it should be fine for the approval and tx to have the same expiration no?